docs/guide/learning/faq.md

```diff
@@ -8,591 +8,47 @@ Answers may contain links to other Vue community pages, external articles and bl
-
-You can leverage Webpack's file resolution and aliases inside the import statement, which means you can use `~` to access `node_modules` packages `@import "~animate-css"`, or `@` alias to access the `src` folder - `@import "@/components/something"`.
-
-### How do scope styles work
-
-When the `scoped` attribute is applied to the `style` tag, CSS rules defined within it will only apply to the current component. This is done by adding randomly named data attributes to each element, that styled by the rules of that `style` tag. This is done by Vue CLI via PostCSS, behind the scenes while building the application, and does not require any further configuration from the developer.
-
-<useful-links>
-<useful-links-section title="Official">
-
-* [Scoped CSS](https://vue-loader.vuejs.org/guide/scoped-css.html)
-
-</useful-links-section>
-</useful-links>
-
-### Can I mix both scoped and global styles in one file
-
-Yes, you can mix them, just define a second style tag.
- 
-<useful-links>
-<useful-links-section title="Official">
-
- * [Mixing Local and Global Styles](https://vue-loader.vuejs.org/guide/scoped-css.html#mixing-local-and-global-styles)
-
-</useful-links-section>
-</useful-links>
-
-### What is the difference between scope styles and css modules
-
-Scoped styles automatically change the css rules to match the auto generated data-attributes on each element that is being styled. This is done without any extra effort from the developer.
-
-CSS Modules allow for, again, uniquely generated classes, that are accessible from within the Vue Template, via a special `$style` property. One major benefit of Modules is that you can export css variables to JavaScript.
-
-<useful-links>
-<useful-links-section title="Tutorials">
-
-* [Vue.js - Scoped Styles vs CSS Modules](https://www.netguru.com/codestories/vue.js-scoped-styles-vs-css-modules)
-
-</useful-links-section>
-</useful-links>
-
-### How can I target children with scoped styles
-
-Using the `deep` selector in your sass files.
-
-<useful-links>
-<useful-links-section title="Tutorial">
-
-[How to Target Child Components with Scoped CSS in Vue](https://bambielli.com/til/2018-08-19-how-to-target-child-components-with-scoped-css-in-vue/#)
-
-</useful-links-section>
-</useful-links>
-
-
-### How to do CSS in JS with Vue
-
-The most popular library for Vue is [vue-styled-components](https://github.com/styled-components/vue-styled-components), which is a port of the original `styled-components` library.
+<!--
+  Contributors:
+
+    The FAQs are now stored outsode of the docs/ directory, in faqs/
+
+    FAQS:   https://github.com/dobromir-hristov/vuecommunity/tree/master/faqs
+    README: https://github.com/dobromir-hristov/vuecommunity/tree/master/faqs/README.md
+-->
+
+<faq-category v-for="(faqs, categoryName) in faqsByCategory" :name="categoryName">
+  <faq-item v-for="faq in faqs" v-bind="faq" />
+</faq-category>
+
+<script>
+export default {
```

Comment:
I dont really like this in the md file directly :/

Reply:
It can certainly be done elsewhere, I'll take care of that.